homeassistant/components/nam/strings.json

```diff
@@ -39,15 +39,125 @@
     }
   },
   "entity": {
+    "button": {
+      "restart": {
+        "name": "[%key:component::button::entity_component::restart::name%]"
+      }
+    },
     "sensor": {
-      "caqi_level": {
+      "bme280_humidity": {
+        "name": "BME280 humidity"
```

Comment:
Should the name of the sensor be capitalized?
`<device name> BME280 humidity` or `<device name> BME280 Humidity`?

Reply:
As you did it now, is the proper sentence style we are looking for, thanks 👍

Reply:
Thanks